src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator started its work");

```

Comment:
The message in the `doWork` method should be more informative by indicating the type of `Machine` that is working, such as 'Excavator'.